packages/core/tests/lcp.test.ts

```diff
@@ -19,35 +20,57 @@ describe('LCP calculate correctly', () => {
   it('correct 1', async () => {
     vol.fromJSON({ '/Users/Someone/project-a/src': null });
 
-    const result = await calcLongestCommonPath([
-      '/Users/Someone/project-a/src/helpers',
-      '/Users/Someone/project-a/src',
-      '/Users/Someone/project-a/src/utils',
-    ]);
-    expect(result).toBe('/Users/Someone/project-a/src');
+    if (path.sep === '/') {
```

Comment:
```suggestion
    if (os.platform() !== 'win32') {
```

Could make the code more clear.